php/bin/cli.php

```diff
@@ -10,8 +10,10 @@
 use Commands\Objects\UpdateCommand;
 use Symfony\Component\Console\Application;
 
-$dotenv = Dotenv\Dotenv::createImmutable(__DIR__.'/../');
-$dotenv->load();
+if (file_exists(__DIR__.'/../.env')) {
```

Comment:
What should happen when file doesn't exist?

Reply:
It should not loading